parser/src/cfg/defcfg.rs

```diff
@@ -238,6 +242,18 @@ pub fn parse_defcfg(expr: &[SExpr]) -> Result<CfgOptions> {
                             cfg.windows_interception_mouse_hwid = Some(hwid_slice?);
                         }
                     }
+                    "macos-dev" => {
+                        #[cfg(target_os = "macos")]
```

Comment:
```suggestion
                        #[cfg(any(target_os = "macos", target_os = "unknown"))]
```
similar to above (https://github.com/jtroo/kanata/pull/652/files#r1419935217)